docs/react-testing-library/setup.mdx

```diff
@@ -12,7 +12,7 @@ there are some things you can do when configuring your testing framework to
 reduce some boilerplate. In these docs we'll demonstrate configuring Jest, but
 you should be able to do similar things with
 [any testing framework](#using-without-jest) (React Testing Library does not
-require that you use Jest).
+require that you use Jest). For example, using [React Testing Library with Vitest](https://www.robinwieruch.de/vitest-react-testing-library/) is a popular alternative to using it with Jest.
```

Comment:
Hi, the linked article is good and explains how to do it.
But I think we prefer to "inline" that content into our docs instead of referring to an external website/blog.
For example, similar to the [Svelte Testing Library docs](https://testing-library.com/docs/svelte-testing-library/setup#vitest).

Before closing this PR I'll wait until a second maintainer gives a review.

Reply:
I agree.. I think that blog posts tend to get outdated quite quickly and we aim to keep stuff inside or docs so we'll have the ability to update them :)

Reply:
I closed this, but feel free to open a PR with a "copy-paste" of the linked article.